packages/core/src/Modules/Cashier/Components/Form/account-transfer-form.jsx

```diff
@@ -324,7 +324,7 @@ const AccountTransferForm = ({
                                             className='cashier__input dc-input--no-placeholder account-transfer__input'
                                             type='text'
                                             label={localize('Amount')}
-                                            error={touched.amount && errors.amount}
+                                            error={(touched.amount && errors.amount) || ''}
```

Comment:
(Question) Why are we falling back to a truthy value, when `errors.amount` is undefined?

Reply:
for this case, we faced `Warning: Failed prop type: Invalid prop `error` of type `boolean` supplied to `Input`, expected `string`.
` error. that's why I pass `an empty string`.